susePublicCloudInfoClient/Makefile

```diff
@@ -7,7 +7,9 @@ files = Makefile README.md LICENSE pint setup.py
 
 nv = $(shell rpm -q --specfile --qf '%{NAME}-%{VERSION}\n' *.spec)
 verSpec = $(shell rpm -q --specfile --qf '%{VERSION}' *.spec)
-verSrc = $(shell cat lib/susepubliccloudinfoclient/VERSION)
+verSrc = $(shell cat lib/susepubliccloudinfoclient/version.py | \
+	grep __version__ | awk -F\' '{ print $$2 }' | tr -d '\n')
+
```

Comment:
I didn't want to fire up the python interpreter to figure out the version, although I have to admit that awk is pretty archaic, what can I say, I am old ;)

Will fix the name of the variable, want to stick as close to pep 8 as possible.